src/components/text-editor/text-editor.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+    Component,
+    Element,
+    Event,
+    EventEmitter,
+    State,
+    h,
+} from '@stencil/core';
+import { EditorState } from 'prosemirror-state';
+import { EditorView } from 'prosemirror-view';
+import { Schema, DOMParser } from 'prosemirror-model';
+import { schema } from 'prosemirror-schema-basic';
+import { exampleSetup } from 'prosemirror-example-setup';
+
+/**
+ * The `limel-text-editor` component wraps the toast-ui react editor. This editor
+ * allows a rich text editing experience with markdown support.
```

Comment:
We should update this description before merging.

Reply:
⚡